changelog.d/11267.doc

```diff
@@ -0,0 +1 @@
+Correct synapse install command for FreeBSD.
```

Comment:
```suggestion
Update Synapse install command for FreeBSD as the package is now prefixed with `py38`. Contributed by @itchychips.
```
Noob talking here, so correct me if this sounds rubbish. Would you like to be mentioned in the changelog? :)